src/MeshIntegrationService/RetrieveMeshFile/local.settings.json.template

```diff
@@ -0,0 +1,18 @@
+{
+    "IsEncrypted": false,
+    "Values": {
+        "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+        "FUNCTIONS_WORKER_RUNTIME": "dotnet-isolated",
+        "MeshApiBaseUrl" : "http://localhost:8700/messageexchange",
+        "BSSMailBox": "X26ABC1",
+        "bssfolder_STORAGE":"UseDevelopmentStorage=true",
```

Comment:
Can we use the AzureWebJobsStorage environment variable instead of creating an additional variable with the same value?